tableauserverclient/server/endpoint/resource_tagger.py

```diff
@@ -0,0 +1,33 @@
+from .endpoint import Endpoint
+from .. import RequestFactory
+from ...models.tag_item import TagItem
+import logging
+import copy
+
+logger = logging.getLogger('tableau.endpoint')
+
+
+class ResourceTagger(Endpoint):
+    # Add new tags to resource
+    def _add_tags(self, baseurl, resource_id, tag_set):
```

Comment:
better to not have these prefixed with _ and to preface the entire class with _ to signify that this isn't for end-user use.